docs/docs/01-quickstart.md

```diff
@@ -301,10 +301,11 @@ Refocus lets you limit the number of requests per user--once the user reaches th
 
 Configure your limits using the following environment variables:
```

Comment:
Now that the env var names are in line with the actual express limiter module, also add a link to that module here so people can get more info if they need it.